### PR TITLE
Finish

### PR DESCRIPTION
**Pull request by ALPANNA-GH**
1. initiate 2nd milestone of the Portfolio project
2. setup linter & git
3. some enhancements for the readme file
4. structure HTML wireframe design according to Figma template-2
5. place elements in their positions using grid and flexbox
6. style all elements according to temp-2 
7. adjust positions, fonts, and colour.
8. fix gaps and alignments.
9. refactoring few code lines

### CODE REVIEW

**Comment by Kamba56** [comment]
Hi @ALPANNA-GH 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly fix stylelint errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -47,6 +46,188 @@ <h3>I'm a Software Developer</h3>
...
+          A daily selection of privately personalized reads;
+          no accounts or sign-ups required. Has been the industry's standard.
+        </p>
+        <ul>
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button type="button">See project</button>
```
Comment (ShadyShawkat) [inline index.html:157]: - [ ] Kindly make sure that the buttons have the appropriate interaction styles as per the design.

image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @ALPANNA-GH, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @ALPANNA-GH , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fixed reviewer suggestions ✔️
- Well STractured Code ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**